### ISSUE TITLE
PHP 5.6 : iconv_set_encoding

### ISSUE THREAD

**Issue by ctadlock** [amzn/amazon-payments-magento-plugin#232]
There are a couple usages of `iconv_set_encoding` which have been deprecated in PHP 5.6.

* `MarketplaceWebServiceSellers/Client.php`
* `MarketplaceWebServiceSellers/KeycheckClient.php`

```
        if (PHP_VERSION_ID < 50600) {
            iconv_set_encoding('output_encoding', 'UTF-8');
            iconv_set_encoding('input_encoding', 'UTF-8');
            iconv_set_encoding('internal_encoding', 'UTF-8');
        } else {
            ini_set('output_encoding', 'UTF-8');
            ini_set('input_encoding', 'UTF-8');
            ini_set('default_charset', 'UTF-8');
        }
```

**Comment by ngrices** [comment]
@ctadlock Want to submit a pull request?

**Comment by ctadlock** [comment]
No. I am not an expert in your codebase, and have no intention of becoming one. It's your codebase and you are the expert; therefore you are the most qualified to make the appropriate changes. As a consumer of your product I identified an issue and tracked it down in detail along with a proposed solution. Is my solution accurate? Is it enough? Are there other related issues? I dont know, but you would.

**Comment by gbear** [comment]
When #219 (keycheck removed/replaced) no longer need the library causing this 5.6 incompatibility issue, so lib likely can be removed also resolving this ticket.

**Comment by jonahellison** [comment]
Resolved in 1.4 branch.
